src/ConstraintViolationException.php

```diff
@@ -28,24 +28,14 @@ public function isClientSafe(): bool
         return true;
     }
 
-    /**
-     * Returns string describing a category of the error.
-     *
-     * Value "graphql" is reserved for errors produced by query parsing or validation, do not use it.
-     */
-    public function getCategory(): string
-    {
-        return 'Validate';
-    }
-
     /**
      * Returns the "extensions" object attached to the GraphQL error.
      *
      * @return array<string, mixed>
      */
     public function getExtensions(): array
     {
-        $extensions = [];
+        $extensions = ['category' => 'Validate'];
```

Comment:
To keep tests running and for backwards compatibility i added category to extensions manually